Source/Urho3D/Engine/Console.cpp

```diff
@@ -376,14 +376,10 @@ void Console::HandleTextFinished(StringHash eventType, VariantMap& eventData)
         // Send the command as an event for script subsystem
         using namespace ConsoleCommand;
 
-#if URHO3D_CXX11
-        SendEvent(E_CONSOLECOMMAND, P_COMMAND, line, P_ID, static_cast<Text*>(interpreters_->GetSelectedItem())->GetText());
-#else
         VariantMap& newEventData = GetEventDataMap();

```

Comment:
I was the one adding the sample usage there in the first place to demonstrate how it would be done in the future when we have the new standard enabled. Check the commit history and also the rather heated discussion over variadic template implementation in the issue tracker. I didn't change all the rest because I don't want to peppered the code with ifdef/endif everywhere. But to me, now it is the time to rewrite everything and to remove all the old code that look like boilerplate.